.appveyor.yml

```diff
@@ -104,7 +95,7 @@ artifacts:
     type: zip
 
 on_finish:
-  - pip install codecov
+  - conda install codecov
```

Comment:
Conda forge is actually configured by the environment, so everything is conda-forge. The earlier instance of installing win32 could actually remove the explicit `-c conda-forge`

https://ci.appveyor.com/project/matplotlib/matplotlib/builds/45337448/job/aie7c1u8m0pf3nf6#L1893

Reply:
Where does conda say that the channel config from an environment.yml is preseved? I just tested with a minimal
```
name: cftest
channels:
    - conda-forge
dependencies:
    - python=3.7
```
and when I do a `conda install numpy in that environment, `conda` wants to install from the main channel.

Reply:
In that case, its probably because we prepend conda-forge to the base channels list before creating the environment:

https://github.com/matplotlib/matplotlib/blob/e2131bb26e9c60bbb823f74dbb5e6de03ba43ee0/.appveyor.yml#L49

Reply:
Ah, that should be it.